montblanc/impl/biro/v2/loaders/loaders.py

```diff
@@ -34,6 +38,23 @@ def load(self, solver, **kwargs):
 
         na, nbl, ntime, nchan = solver.na, solver.nbl, solver.ntime, solver.nchan
 
+	uvw_order = kwargs.get('uvw_order', ORDER_OTHER)
+        if uvw_order not in VALID_ORDERINGS:
```

Comment:
@marziarivi Remove this check here, but re-add it at the end of the else statement later.